danger-kotlin-library/src/test/kotlin/systems/danger/kotlin/GitKtxTest.kt

```diff
@@ -0,0 +1,77 @@
+package systems.danger.kotlin
+
+import junit.framework.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import systems.danger.kotlin.shell.ShellExecutor
+import systems.danger.kotlin.shell.ShellExecutorFactory
+
+/**
+ * Tests for [Git] extensions
+ */
+internal class GitKtxTest {
+
+    class MockShellExecutor : ShellExecutor {
+
+        private val diffCommandOutput = """
+        0       1       features/search/build.gradle
+        3       10      features/search/src/main/java/com/sampleapp/search/di/RepositoryModule.kt
+        2       4       features/search/src/main/java/com/sampleapp/search/model/ApiInteractor.kt
+        2       4       features/search/src/main/java/com/sampleapp/search/model/RecommendedPublishersRepository.kt
+        1       3       features/search/src/main/java/com/sampleapp/search/model/RecommendedTopicsRepository.kt
+    """.trimIndent()
```

Comment:
```kotlin
        private val diffCommandOutput = """
        0       1       features/search/build.gradle
        3       10      features/search/src/main/java/com/sampleapp/search/di/RepositoryModule.kt
        2       4       features/search/src/main/java/com/sampleapp/search/model/ApiInteractor.kt
        2       4       features/search/src/main/java/com/sampleapp/search/model/RecommendedPublishersRepository.kt
        1       3       features/search/src/main/java/com/sampleapp/search/model/RecommendedTopicsRepository.kt
    """.trimIndent()
```
The """  is not in line with the private, that looks odd

```suggestion
        private val diffCommandOutput = """
        0       1       features/search/build.gradle
        3       10      features/search/src/main/java/com/sampleapp/search/di/RepositoryModule.kt
        2       4       features/search/src/main/java/com/sampleapp/search/model/ApiInteractor.kt
        2       4       features/search/src/main/java/com/sampleapp/search/model/RecommendedPublishersRepository.kt
        1       3       features/search/src/main/java/com/sampleapp/search/model/RecommendedTopicsRepository.kt
        """.trimIndent()
```